src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx

```diff
@@ -0,0 +1,143 @@
+  * **Uninstrumented externals**: These are uninstrumented services that you may or may not own.
+
+## How performance metric charts work [#understand-data]
+
+On the opening page of external services, you see these performance metric charts: **Response time**, **Throughput**, and **Error rate**. For APM agents, these initial performance charts are populated by metric data, while for OpenTelemetry, the initial values are populated by sampled data. As you drill below the initial page–whether it's OpenTelemetry or APM agents–each child page is populated by sampled data. This means that if you're not seeing the data you expect on pages that show sampled data, you may need to increase your [sampling](/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup#adjust-sampling).
+
+The performance metric charts always reflect the data from the page you're viewing, but the set of metrics displayed changes as you drill below the opening page. Here's what you need to know about these metrics:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Metric
+      </th>
+      <th>
+        Screen
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+      <tr>
+        <td>
+          Response Time
+        </td>
+        <td>
+          All screens
+        </td>
+        <td>
+          The average duration of sampled calls between two services in the initial view, or between transactions in the other views. How well this represents actual system performance depends on the effective sample rate. This metric is displayed on each level as you drill down.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Throughput
+        </td>
+        <td>
+          Opening screen
+        </td>
+        <td>
+          ??
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Error rate
+        </td>
+        <td>
+          Opening screen
+        </td>
+        <td>
+          ??
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Traced call count
+        </td>
+        <td>
+          Drill-down pages
+        </td>
+        <td>
+          Represents the number of sampled calls we have for a given path between two services or transactions. This is lower than the total throughput, unless you are sampling 100% of your requests.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Traced error count
+        </td>
+        <td>
+          Drill-down pages
+        </td>
+        <td>
+          The number of sampled calls between two services or transactions that had errors.
+        </td>
+      </tr>
+  </tbody>
+</table>
+
+## The external services table [#table-view]
+
+As a companion to the map view, the table view lists all the related services in a columnar format. When you click **List** in the upper-right of the page, you see the same services from the map view:
```

Comment:
I don't think you need a dash "upper right"

Reply:
@barbnewrelic, good catch!